docs/content/latest/deploy/multi-dc/async-replication.md

```diff
@@ -182,7 +182,47 @@ When both universes use different certificates, you need to store the certificat
 
 ## Bootstrapping a sink cluster
```

Comment:
We should also add a note earlier in this page under the regular setup flow, where we mention that if you already have existing data in your table, then you need to follow the bootstrap proces

Reply:
Added